pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1705,8 +1705,8 @@ func TestPreempt(t *testing.T) {
 			}
 
 			// Call preempt again and make sure it doesn't preempt any more pods.
-			node, err = pl.preempt(context.Background(), state, test.pod, make(framework.NodeToStatusMap))
-			if err != nil {
+			node, status = pl.preempt(context.Background(), state, test.pod, make(framework.NodeToStatusMap))
+			if !status.IsSuccess() {
 				t.Errorf("unexpected error in preemption: %v", err)
```

Comment:
ditto, this is now logging a different error

Reply:
done.